autogluon/scheduler/fifo.py

```diff
@@ -455,6 +455,15 @@ def get_best_config(self):
         """
         return self.searcher.get_best_config()
 
+    def get_best_task_id(self):
+        """Get the task id that results in the best configuration/best reward"""
```

Comment:
Docstring should clarify that earliest `task_id` will be returned in the case of ties.